src/mds/MDBalancer.cc

```diff
@@ -162,7 +162,10 @@ mds_load_t MDBalancer::get_load(utime_t now)
   ifstream cpu("/proc/loadavg");
   if (cpu.is_open())
     cpu >> load.cpu_load_avg;
-
+  else
+    derr << "input file '/proc/loadavg' not found" << dendl;
+  cpu.close();  
```

Comment:
The close() would go inside the is_open() conditional, calling close on a non-open ifstream is an error.